packages/components/forms/src/base-input/BaseInput.tsx

```diff
@@ -57,14 +57,11 @@ function _BaseInput<E extends React.ElementType = typeof DEFAULT_TAG>(
   const handleFocus = useCallback(
     (e: FocusEvent<HTMLInputElement | HTMLTextAreaElement>) => {
       e.persist();
```

Comment:
```suggestion
```

Reply:
we don't need it as well? then we don't need custom `handleFocus` at all too

Reply:
You're right, I would remove it completely.